airflow/config_templates/config.yml

```
@@ -1358,6 +1358,14 @@
       example: "/path/to/my_html_content_template_file"
       default: ~
       see_also: ":doc:`Email Configuration </howto/email-config>`"
+    - name: from_email
+      description: |
+        Email address that will be used as sender address.
+        It can either be raw email or the complete address in a format ``Sender Name <sender@email.com>``
+      version_added: 2.2.0
+      type: string
+      example: "\"Airflow <airflow@example.com>\""
```

```suggestion
      example: "Airflow <airflow@example.com>"
```

I don't think the extra quotes are needed?

---

Also this unfortunately missed 2.2.0 so we need to change `version_added` to 2.3